### ISSUE TITLE
Cannot allow self signed certificates with current `httpClient` access level

### ISSUE THREAD

**Issue by DavidBrunow** [malcommac/GitLabSwift#3]
I'm using this library with an internal GitLab enterprise instance that is using a self-signed certificate. Without adjusting the accessibility of the `GLApi`s `httpClient` I cannot set the security on that client to `.acceptSelfSigned`. Here is where I make that change for our use: https://github.com/malcommac/GitLabSwift/blob/main/Sources/GitLabSwift/Core/GLApi.swift#L76

I don't think that the best answer is to make that `httpClient` public but some way to influence the security would be nice.
